pkg/dynamic-config/dynamic_context.go

```diff
@@ -204,7 +204,10 @@ func (d *DynamicContext) KeyAndSecretFlags(cmd *cobra.Command) (string, string,
 	}
 
 	if apiKey == "" && apiSecret != "" {
-		return "", "", errors.NewErrorWithSuggestions(errors.PassedSecretButNotKeyErrorMsg, errors.PassedSecretButNotKeySuggestions)
+		return "", "", errors.NewErrorWithSuggestions(
+			"no API key specified",
+			"Use the `--api-key` flag to specify an API key.",
+		)
```

Comment:
Are we putting the error and suggestion on different lines for all or some of the calls to `NewErrorWithSuggestions`?

Reply:
Just the ones that are really long if we don't